src/cranbtree.c

```diff
@@ -41,6 +41,12 @@
 #include "lib/update.c"
 #include "lib/clone.c"
 
+/*
+ *macros
+ */
+
```

Comment:
This should not be defined here. Instead, it should be defined in include/cranbtree.h. However, I have already defined it before opening the issue. Take a look at include/cranbtree.h